.packit.yaml

```diff
@@ -188,3 +199,4 @@ jobs:
       - fedora-branched
       - epel-9
       - epel-10
+
```

Comment:
Pre Commit is failing due to trailing whitespaces : Please run `pre-commit run --all-files` locally and it will fix automatically.